src/script/miniscript.h

```diff
@@ -189,11 +189,11 @@ inline consteval Type operator""_mst(const char* c, size_t l)
 using Opcode = std::pair<opcodetype, std::vector<unsigned char>>;
 
 template<typename Key> class Node;
-template<typename Key> using NodeRef = std::unique_ptr<Node<Key>>;
+template<typename Key> using NodeRef = Node<Key>; // <- removed in next commit
```

Comment:
👍 - if you touch again, consider adding a `TODO` prefix to make sure we get a warning from the linters or the IDE if we accidentally leave them there